meson.build

```diff
@@ -300,7 +301,14 @@ else
 endif
 
 if want_libfuzzer
```

Comment:
Is want_libfuzzer supposed to be true when want_ossfuzz_without_fuzzing_engine is true?

Reply:
`want_libfuzzer` and `want_ossfuzz*` can't be used together. `meson` will bail out if both `want_libfuzzer` and `want_ossfuzz` is set to `true`.